### PR TITLE
6. Consuming get methods

### PR DESCRIPTION
**Pull request by lvzh90**
6. Consuming get methods

### CODE REVIEW

File: test/GithubApi.Repositories.test.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          repositories = response.body;
+          repository = repositories.find(repo => repo.name === expectedRepository);
+
+          assert.exists(repository);
+          expect(repository.full_name).to.equal('aperdomob/jasmine-awesome-report');
+          expect(repository.private).to.equal(false);
+          expect(repository.description).to.equal('An awesome html report for Jasmine');
+        }));
+
+      describe(`• Download ${expectedRepository} repository`, () => {
```
Comment (germandavid85) [inline test/GithubApi.Repositories.test.js]: let's avoid these special characters like the bullet point, the same for lines 10, 26, 51 and 70

File: test/GithubApi.Repositories.test.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+          const format = {
+            name: 'README.md',
+            path: 'README.md',
+            sha: '9bcf2527fd5cd12ce18e457581319a349f9a56f3'
+          };
+
+          it('Searching...', () => agent.get(`${repository.url}/contents`)
+            .then((response) => {
+              files = response.body;
```
Comment (germandavid85) [inline test/GithubApi.Repositories.test.js]: files can be a `const` and can be defined here

File: test/GithubApi.Repositories.test.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        expect(userSession.company).equal('PSL');
+        expect(userSession.location).equal('Colombia');
+      }));
+
+    describe('• Get repository list', () => {
+      const expectedRepository = 'jasmine-awesome-report';
+      let repositories;
+      let repository;
+
+      it(`${expectedRepository} repository`, () => agent.get(userSession.repos_url)
```
Comment (germandavid85) [inline test/GithubApi.Repositories.test.js]: you can separate the API call from the assertion part by moving it into a `before` like this

```js
before(() =>
  agent.get(userSession.repos_url)
      .then((response) => {
          repositories = response.body;
          repository = repositories.find(repo => repo.name === expectedRepository);
    });
)

it(`${expectedRepository} repository`, () => {
    assert.exists(repository);
    expect(repository.full_name).to.equal('aperdomob/jasmine-awesome-report');
    expect(repository.private).to.equal(false);
    expect(repository.description).to.equal('An awesome html report for Jasmine');
});
```

that way we can separate the preconditions from the validations and keep the code more organized

**Review by germandavid85 — CHANGES_REQUESTED** [review]
a couple of small comment

File: test/GithubApi.Repositories.test.js (hunk 1 above)
Comment (lvzh90) [reply test/GithubApi.Repositories.test.js]: Hecho

File: test/GithubApi.Repositories.test.js (hunk 2 above)
Comment (lvzh90) [reply test/GithubApi.Repositories.test.js]: Hecho

File: test/GithubApi.Repositories.test.js (hunk 3 above)
Comment (lvzh90) [reply test/GithubApi.Repositories.test.js]: Hecho

**Review by germandavid85 — APPROVED** [review]
LGTM 👍

**Pull request merged**